platform/pulpcore/app/views/status.py

```diff
@@ -0,0 +1,35 @@
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from pulpcore.app.serializers.task import WorkerSerializer
+import status
+
+
+class StatusView(APIView):
+    """
+    Returns status information about the application
+    """
+    authentication_classes = []
+    permission_classes = []
+
+    def get(self, request, format=None):
+        """
+        Returns app information including the version, known workers, database connection status,
+        and messaging connection status
+        """
+        version = {'pulpcore_version': status.get_version()}
+
+        workers = status.get_workers()
+        context = {'request': request}
+        worker_status = [WorkerSerializer(worker, context=context).data for worker in workers]
+
+        broker_status = {'connected': status.get_broker_conn_status()}
+        db_status = {'connected': status.get_db_conn_status()}
+
+        response = {
```

Comment:
I wonder if this should be done by a Serializer. Perhaps that would better enable DRF to introspect the returned structure, but I'm fuzzy on those details of DRF. Thoughts?

@asmacdo  ?